packages/cli/cli/src/cli.ts

```diff
@@ -152,6 +154,8 @@ async function tryRunCli(cliContext: CliContext) {
     addTestCommand(cli, cliContext);
     addUpdateApiSpecCommand(cli, cliContext);
     addUpgradeGeneratorCommand(cli, cliContext);
+    addGetOrganizationCommand(cli, cliContext);
+    addGeneratorListCommand(cli, cliContext);
```

Comment:
@armandobelardo are you cool to start a `cliV2.ts` file and move these into that ?

Reply:
no functional difference but will allow us to know to keep these commands / they are agreed upon for v2 already

Reply:
Moved them over and referenced them here